development/staging.md

```diff
@@ -0,0 +1,32 @@
+# Staging Environment
+
+We have a staging environment running on AWS which is a scaled down replica of FlowForge Cloud.
+
+It uses the domain flowforge.dev
+
+## AWS Account
+
+It uses a separate AWS account ending in ..9937
+
+Ben or ZJ can provision a user account for this account.
+
+The services are running in EU-West-1.
+
+## Nodes
+The staging environment uses one node running on a t2.micro for the management app and a pair of t2.micro nodes for the projects cluster.
```

Comment:
t2.small

Reply:
doh I thought we went nano>micro, corrected